src/main/java/com/fasterxml/jackson/databind/deser/BeanDeserializer.java

```diff
@@ -467,6 +467,10 @@ protected Object _deserializeUsingPropertyBased(final JsonParser p, final Deseri
             // regular property? needs buffering
             SettableBeanProperty prop = _beanProperties.find(propName);
             if (prop != null) {
+                if (prop.isIgnorable() && _beanType.isRecordType()) {
```

Comment:
Was there a problem here, to require additional processing? And/or would this work even for POJO types?